packages/babel-parser/test/helpers/runFixtureTests.js

```diff
@@ -41,6 +41,11 @@ class FixtureError extends Error {
   }
 }
 
+// https://github.com/estree/estree/blob/master/es2020.md#bigintliteral
+function bigIntReplacer(key, value) {
```

Comment:
`BigInt`s can't be serialized. In the ESTree spec, `BigInt` values are set to `null` when the type isn't supported by an environment, which seemed like a reasonable behavior. Another option we have is to use `BigInt.prototype.toString()` - thoughts on this? Note that with the latter, `1n` will become just `1`, so if we do that, we might want to add the `n` to it. Not sure if this would have some negative side effects though, since we're altering the type.